platform/.env.example

```diff
@@ -1,4 +1,6 @@
 NEXT_PUBLIC_ORIGINAL_PLATFORM=https://platform.airqo.net/
 NEXT_PUBLIC_API_TOKEN=token_value
 NEXT_PUBLIC_API_BASE_URL=https://platform.airqo.net/api/v1
-SECRET_KEY_1=secret_key_1_value
\ No newline at end of file
+SECRET_KEY_1=secret_key_1_value
+NEXT_PUBLIC_GRIDS=64b7baccf2b99f00296acd59,64b7ac8fd7249f0029feca80,64d635f18f492b0013406c46,64c90174448a63001e3a0b45
```

Comment:
Hi @FaithDaka , please use ENVs instead of directly inputing things here. These ENVS were already created in both PRODUCTION and STAGING so just call them accordingly and also update your local .env file accordingly.

Reply:
This is the documentation @Baalmart